plugins/MauticCitrixBundle/Command/SyncCommand.php

```diff
@@ -0,0 +1,165 @@
+<?php
+/**
+ * @copyright   2014 Mautic Contributors. All rights reserved
+ * @author      Mautic
+ *
+ * @link        http://mautic.org
+ *
+ * @license     GNU/GPLv3 http://www.gnu.org/licenses/gpl-3.0.html
+ */
+
+namespace MauticPlugin\MauticCitrixBundle\Command;
+
+use MauticPlugin\MauticCitrixBundle\Entity\CitrixEventTypes;
+use MauticPlugin\MauticCitrixBundle\Helper\CitrixHelper;
+use MauticPlugin\MauticCitrixBundle\Helper\CitrixProducts;
+use MauticPlugin\MauticCitrixBundle\Model\CitrixModel;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * CLI Command : Synchronizes registrant information from Citrix products
+ *
+ * php app/console citrix:sync [--product=webinar|meeting|assist|training [--id=%productId%]]
+ *
+ */
+class SyncCommand extends ContainerAwareCommand
+{
+    /** @var  CitrixModel */
+    protected $citrixModel;
+
+    /**
+     * SyncCommand constructor.
+     * @param null $name
+     * @throws \Symfony\Component\Console\Exception\LogicException
+     * @throws \Symfony\Component\DependencyInjection\Exception\ServiceCircularReferenceException
+     * @throws \Symfony\Component\DependencyInjection\Exception\ServiceNotFoundException
+     */
+    public function __construct($name = null)
+    {
+        parent::__construct($name);
+        $container = CitrixHelper::getContainer();
+        $factory = $container->get('mautic.model.factory');
+        $this->citrixModel = $factory->getModel('citrix.citrix');
+    }
+
+    /**
+     * {@inheritdoc}
+     * @throws \Symfony\Component\Console\Exception\InvalidArgumentException
+     */
+    protected function configure()
+    {
+        $this->setName('citrix:sync')
```

Comment:
Let's keep this congruent with Mautic's commands by prefixing with `mautic` so `mautic:citrix:sync`